competing-consumers/DeploymentTemplate/CompetingConsumers.json

```diff
@@ -0,0 +1,239 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sites_competingconsumers_name": {
+            "defaultValue": "competingconsumers",
+            "type": "String"
+        },
+        "components_competingconsumers_name": {
+            "defaultValue": "competingconsumers",
+            "type": "String"
+        },
+        "serverfarms_asp_competingconsumers_name": {
+            "defaultValue": "asp-competingconsumers",
+            "type": "String"
+        },
+        "namespaces_sbcompetingconsumers_name": {
+            "defaultValue": "sbcompetingconsumers",
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.ServiceBus/namespaces",
+            "apiVersion": "2018-01-01-preview",
+            "name": "[parameters('namespaces_sbcompetingconsumers_name')]",
+            "location": "[resourceGroup().location]",
+            "sku": {
+                "name": "Premium",
+                "tier": "Premium",
+                "capacity": 1
+            },
+            "properties": {
+                "zoneRedundant": false
+            }
+        },
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "2018-02-01",
+            "name": "[parameters('serverfarms_asp_competingconsumers_name')]",
+            "location": "[resourceGroup().location]",
+            "sku": {
+                "name": "S1",
+                "tier": "Standard",
+                "size": "S1",
+                "family": "S",
+                "capacity": 1
+            },
+            "kind": "app",
+            "properties": {
+                "perSiteScaling": false,
+                "maximumElasticWorkerCount": 1,
+                "isSpot": false,
+                "reserved": false,
+                "isXenon": false,
+                "hyperV": false,
+                "targetWorkerCount": 0,
+                "targetWorkerSizeId": 0
+            }
+        },
+        {
+            "type": "Microsoft.ServiceBus/namespaces/AuthorizationRules",
+            "apiVersion": "2017-04-01",
+            "name": "[concat(parameters('namespaces_sbcompetingconsumers_name'), '/RootManageSharedAccessKey')]",
```

Comment:
suggestion: just create two, one for Listen and one for Send, then you can assign to the various jobs.  Unless you think that will complicate the readme too much?  -- Otherwise, a root key, IIRC is already created for you and you wouldn't need to include this resource.